ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/example_code/ad_utils.py

```diff
@@ -63,28 +73,102 @@ def _create_model_(self, project_id, compartment_id, data_asset_id,
             f"Error creating model: {create_model_response.text}"
         return create_model_response.data.id
 
+    def create_detect_anomalies_job(self, compartment_id, model_id,
+                                    data_asset_detail,
+                                    output_path) -> DetectAnomalyJob:
+
+        input_details = ObjectListInputDetails(
+            object_locations=[ObjectLocation(
+                namespace_name=data_asset_detail['namespace'],
+                bucket_name=data_asset_detail['bucket'],
+                object_name=data_asset_detail['object'])])
+        output_details = ObjectStoreOutputDetails(
+            namespace_name=data_asset_detail['namespace'],
+            bucket_name=output_path,
+            prefix="model1")
+        create_detect_anomaly_job_details = CreateDetectAnomalyJobDetails(
+            compartment_id=compartment_id,
+            display_name="e2e_template_test_job",
+            model_id=model_id,
+            input_details=input_details,
+            output_details=output_details)
+        print(create_detect_anomaly_job_details)
```

Comment:
Please add more context on what is this log so that someone looking at the logs without the context of this script would be able to figure out what object the log is having. Similar comment for other places.